sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/_tests_/e2e/confirmation-of-enrollment.institutions.confirmEnrollment.e2e-spec.ts

```diff
@@ -87,6 +91,58 @@ describe("ConfirmationOfEnrollmentInstitutionsController(e2e)-confirmEnrollment"
     expect(updatedApplication.applicationStatus).toBe(
       ApplicationStatus.Completed,
     );
+
+    // Assert that the sequence control record was created correctly.
+    const sequenceControl = await sequenceControlRepo.findOne({
+      where: {
+        sequenceName: "Full Time_DISBURSEMENT_DOCUMENT_NUMBER",
+        sequenceNumber: firstDisbursementSchedule.documentNumber.toString(),
+      },
+    });
+    expect(sequenceControl).toBeDefined();
+  });
+
+  it("Should allow the COE confirmation when the application is on Enrolment status and all the conditions are fulfilled for Part Time offering.", async () => {
+    // Arrange
+    const application = await saveFakeApplicationDisbursements(
+      appDataSource,
+      {
+        institution: collegeC,
+        institutionLocation: collegeCLocation,
+      },
+      {
+        applicationStatus: ApplicationStatus.Enrolment,
+        offeringIntensity: OfferingIntensity.partTime,
+      },
+    );
+    const [firstDisbursementSchedule] =
+      application.currentAssessment.disbursementSchedules;
+    const endpoint = `/institutions/location/${collegeCLocation.id}/confirmation-of-enrollment/disbursement-schedule/${firstDisbursementSchedule.id}/confirm`;
+    await request(app.getHttpServer())
+      .patch(endpoint)
+      .send({ tuitionRemittanceAmount: 1 })
+      .auth(
+        await getInstitutionToken(InstitutionTokenTypes.CollegeCUser),
+        BEARER_AUTH_TYPE,
+      )
+      .expect(HttpStatus.OK);
+    // Act/Assert
+    // Check if the application was updated as expected.
+    const updatedApplication = await applicationRepo.findOne({
+      select: { applicationStatus: true },
+      where: { id: application.id },
+    });
+    expect(updatedApplication.applicationStatus).toBe(
+      ApplicationStatus.Completed,
+    );
+    // Assert that the sequence control record was created correctly.
+    const sequenceControl = await sequenceControlRepo.findOne({
```

Comment:
Minor but the find one without a select filter will return all the record columns, which would not be used for the assert.
Please consider changing to an exists, as shown below.
```ts
const wasExpectedSequenceControlUsed = await sequenceControlRepo.exists({
  where: {
    sequenceName: "Full Time_DISBURSEMENT_DOCUMENT_NUMBER",
    sequenceNumber: firstDisbursementSchedule.documentNumber.toString(),
  },
});
expect(wasExpectedSequenceControlUsed).toBe(true);
```